docs/reference/reference-training/experiment-config-reference.rst

```diff
@@ -91,7 +100,7 @@ directory name is prefixed by ``<module>`` or used as the module if the ``<modul
 Arbitrary Script
 ================
 
-An arbitrary entrypoint script name.
+Required. An arbitrary entrypoint script name.
```

Comment:
`entrypoint` is a single field, and it's required.  But "Abitrary Script", "Preconfigured Launch Module with Script", "Preconfigured Launch Module with Legacy Trial Definition", and "Legacy Trial Definition" are each different _ways_ of filling out that `entrypoint` required field.  So each different way shouldn't probably comment on if it's required or not, and they should likely be subsections of the `entrypoint` section.

Reply:
resolved by assigning level 4 headings (-------) to: "Arbitrary Script", "Preconfigured Launch Module with Script", "Preconfigured Launch Module with Legacy Trial Definition", and "Legacy Trial Definition"